simulation/model.py

```diff
@@ -57,6 +58,7 @@ def __init__(
             if air_source_heat_pump_price_discount_schedule
             else None
         )
+        self.household_count = household_count
```

Comment:
Instead of putting this in the `__init__` should we make it a `@property` ? Isn't it just:

```
    @property
    def household_count(self):
        return len(self.space.agents)
```
Don't know how this will affect the test below, though.

Reply:
There's no need to re-derive this throughout the simulation though? It's static